### ISSUE TITLE
Add retry ability for contest comparison report/JPanel using since_token

### ISSUE THREAD

**Issue by lane55** [pc2ccs/pc2v9#561]
**Is your feature request related to a problem?** 

Yes, from time to time the API EF is dropped.  If
dropped the report will read from the start of the EF,
rather than continue from the last event processed.

**Feature Description**: 

Add a reconnect using the EF API parameter ?since_token=

**Have you considered other ways to accomplish the same thing?** 

n/a

**Do you have any specific suggestions for how your feature would be ***implemented*** in PC^2?**

Use the method  IRemoteContestAPIAdapter.getRemoteEventFeedInputStream(String token) to start reading the "last" event.

**Additional context**:

**Comment by lane55** [comment]
Blocked by compare contest report  #536 / PR #547

**Comment by johnbrvc** [comment]
Duplicate of #549
